src/Security/Authentication/OpenIdConnect/src/OpenIdConnectHandler.cs

```diff
@@ -276,7 +276,7 @@ public async virtual Task SignOutAsync(AuthenticationProperties properties)
                 Response.Headers[HeaderNames.Pragma] = "no-cache";
                 Response.Headers[HeaderNames.Expires] = HeaderValueEpocDate;
 
-                await Response.Body.WriteAsync(buffer, 0, buffer.Length);
+                await Response.Body.WriteAsync(buffer, Context.RequestAborted);
```

Comment:
It's better not to pass the CT to WriteAsync. The write will fail silently if the client disconnects, but if you pass the CT then it will throw. We don't care if this write succeeds or fails.